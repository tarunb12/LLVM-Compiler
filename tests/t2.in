int main() {
    printf("testing: %.2f", 1.00)
}