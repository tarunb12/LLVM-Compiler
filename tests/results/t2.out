testing: 1.00